main.go

```diff
@@ -50,7 +50,8 @@ func main() {
 	webhook := flag.Bool("webhook", false, "Runs the webhook webserver.")
 	audit := flag.Bool("audit", false, "Runs a one-time audit.")
 	auditPath := flag.String("audit-path", "", "If specified, audits one or more YAML files instead of a cluster")
-	setExitCode := flag.Bool("set-exit-code-on-error", false, "set an exit code of 2 when the audit contains error-level issues.")
+	setExitCode := flag.Bool("set-exit-code-on-error", false, "When running with --audit, set an exit code of 3 when the audit contains error-level issues.")
+	minScore := flag.Int("set-exit-code-below-score", 0, "When running with --audit, set an exit code of 4 when the score is below this threshold (1-100)")
```

Comment:
I don't think it's overly clear that these actually change the output behavior of the audit run when you set these. My expected behavior would be that these only `--set-exit-code-[on-condition]` vs changing the audit reports _and_ change the exit code. Happy to discuss more

Reply:
ran through the code more in depth: https://github.com/reactiveops/polaris/pull/157#discussion_r297996606 -- resolving comment